src/account-info/AccountDetails.js

```diff
@@ -64,9 +65,7 @@ class AccountDetails extends PureComponent<Props> {
           <UserAvatar avatarUrl={getAvatarFromUser(user, realm, AVATAR_SIZE)} size={AVATAR_SIZE} />
         </View>
         <View style={componentStyles.statusWrapper}>
-          <View style={componentStyles.statusSymbolWrapper}>
-            <PresenceStatusIndicator email={user.email} hideIfOffline={false} />
-          </View>
+          <PresenceStatusIndicator style={componentStyles.statusSymbolWrapper} email={user.email} hideIfOffline={false} />
```

Comment:
Since we're no longer using the wrapping `View`, `statusSymbolWrapper` is no longer quite the right name. 🙂

I think just `statusSymbol` would be good—or maybe `statusIndicator` would be better, to match the names we've been using (e.g., `PresenceStatusIndicator`).